### ISSUE TITLE
English / 英语

### ISSUE THREAD

**Issue by AlphaHot** [MLNLP-World/Paper_Writing_Tips#1]
Who can translate this repository into English? Please translate. 😢 
谁能把这个存储库翻译成英语? 请翻译 😭

**Comment by qinlibo-hit** [comment]
@AlphaHot Hi, AlphaHot, thanks for your kind suggestion.  This is a very initial version and we consider introducing an English version in the future. Thanks again. Hope it's helpful for you:)

**Comment by Doragd** [comment]
Typo: There seems to be an extra `s` at the end of the line.
https://github.com/MLNLP-World/Paper_Writing_Tips/blob/e028f19bdf956c76a8e008d6b46c6b3459cbf079/README.md?plain=1#L98

**Comment by qinlibo-hit** [comment]
@Doragd yes, this is a typo. Maybe you can directly give a pull request so that you are in the contributor list:)
